plugins/chat/app/models/concerns/chat/chatable.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
Might be something we want to add in the guide, concerns have to be in their own namespaced folder